src/basic/conf-files.c

```diff
@@ -407,19 +409,33 @@ int conf_file_read(
 
                 log_debug("Reading config from stdin%s", special_glyph(SPECIAL_GLYPH_ELLIPSIS));
```

Comment:
i'd really do the `r = 0` here rather than when declaring the variable. Seems more obvious and harder to get lost as this is updated